src/pydvl/utils/types.py

```diff
@@ -42,6 +46,27 @@ def score(self, x: NDArray, y: NDArray) -> float:
         pass
 
 
+def call_fun_remove_arg(*args, fun: Callable, arg: str, **kwargs):
```

Comment:
What is gained by having this function? Why not leave it in `maybe_remove_arg` since it's only used there?

Reply:
It is required to be pickable. Once the function is wrapped (happens for `seed` and not for `job_id`) it can not be pickled anymore by the default pickle library. To not over-complicate the code by introducing `dill` or `cloudpickle`, I moved the function to the outer-scope.

Reply:
Note that ray depends on cloudpickle, it's pretty much standard. But it's good to avoid dependencies. Then please document the choice in the code, so as to avoid these discussions. Please **always** document decisions which might not be obvious to other readers of the code, because we waste a lot of time otherwise.

Reply:
Done